app/src/main/java/de/westnordost/streetcomplete/screens/user/profile/ProfileViewModel.kt

```diff
@@ -75,37 +68,6 @@ class ProfileViewModelImpl(
     override val biggestSolvedCountCountryStatistics = MutableStateFlow<CountryStatistics?>(null)
     override val biggestSolvedCountCurrentWeekCountryStatistics = MutableStateFlow<CountryStatistics?>(null)
 
-    override var lastShownGlobalUserRank: Int?
```

Comment:
@matkoniecz I removed this because I found it to be a bit complicated or at least not worth the amount of code to keep it in Compose. Now, the badges just always animate, which is nicer anyway. Sure, the sense of progression is lost, but that feature was too well hidden anyway, i.e. it was barely noticeable what changed exactly because the animation was played automatically , so one would have to look really quick anyway to see what the value before was. And making it more prominent would on the other hand usually just look too disruptive. (Don't need a "New achievement unlocked"-type of dialog every time my rank is changed towards the better)

Reply:
After encountering it several times since them I want to say - this was actually a significant improvement in how nice this menu is!